pkg/api/customization/globaldns/action_globaldns.go

```diff
@@ -52,7 +52,9 @@ func (w *Wrapper) ActionHandler(actionName string, action *types.Action, request
 		return fmt.Errorf("GlobalDNS %v has no creatorId annotation", metaAccessor.GetName())
 	}
 	ma := gaccess.MemberAccess{
-		Users: w.Users,
+		Users:     w.Users,
+		GrbLister: w.GrbLister,
+		GrLister:  w.GrLister,
 	}
```

Comment:
There seems to be another instance in  this file where we initialize this struct, can we initialize it with all data as well?

Reply:
So in that method it's only calling a method which doesn't require any of it

Reply:
I guess this is a case where the panic is not caught easily since we are initialising the struct adhoc as needed. But I think it would be good to initialise it with all info available here as well, to prevent any panics in future if code changes.

Reply:
I mean it can be initialized later with all fields if needed, but okay I'll add them here too